deploy/terraform/imdown.tf

```diff
@@ -0,0 +1,38 @@
+provider "aws" {
+  profile = "imdown"
+  region  = "us-west-1"
+}
+
+resource "aws_instance" "imdown" {
+  ami           = "ami-0d705db840ec5f0c5"
+  instance_type = "t2.micro"
+  key_name      = "imdown"
+
+  connection {
+    type        = "ssh"
+    user        = "ubuntu"
+    private_key = file("~/.ssh/imdown.pem")
+    host        = self.public_ip
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "sudo apt-get update",
+      "sudo apt-get -y upgrade",
+      "sudo apt-get -y install mysql-server",
```

Comment:
So eventually we will add commands here to pull our backend and start it up right? Wonder how we should provide proper environment files

Reply:
Yeah I'm actually going to add deployment to this PR so that everything is in one place for full context.
I was gonna use ansible for provisioning (common, easy-to-use automation tool) instead of bash scripts, and systemd to run the service. I think the environment variables should be provided by systemd to discourage the accidental use of `flask run` from SSH.